vxwhatsapp/consumer.py

```diff
@@ -93,9 +94,18 @@ async def process_message(self, message: IncomingMessage):
             await message.reject(requeue=False)
             return
 
-        async with message.process(requeue=True):
+        async with message.process():
             logger.debug(f"Processing outbound message {msg}")
-            await self.submit_message(msg)
+            try:
+                await self.submit_message(msg)
+            except aiohttp.ClientResponseError as e:
+                if e.status > 499:
+                    await message.reject(requeue=True)
+                    return
+
+                capture_exception(e)
```

Comment:
Interesting, I usually just use something like `logger.exception(f"HTTP Error with status {e.status}, retrying")`, didn't know about this direct call to sentry.

Reply:
I saw that but wasn't sure if it was actually going to sentry, and I though `capture_exception` was nice and explicit